### PR TITLE
feat(tracker): support gen 7 and regional dexes

### PR DESCRIPTION
**Pull request by robinjoseph08**
*✲ﾟ*｡✧٩(･ิᴗ･ิ๑)۶*✲ﾟ*｡✧

### api
- [x] add gen 7 pokemon into db
- [x] add gen 7 evolutions into db

### frontend
- [x] add gen 7 to create dex page
- [x] change serebii url
- [x] change which locations to show
- [x] add alola region filter
- [x] add sprites
- [x] get alola form sprites

### to do
- [x] add gen 7 to edit dex page (will be done in the edit page pr)
- [x] region mobile has an extra border when last region is selected. will take care of this later, issue added: https://github.com/robinjoseph08/pokedextracker.com/issues/213
- [x] region - when region filter is missing (not national dex), the padding is off
- [x] profile - possibly adding an indicator for which gen/region a dex is

### CODE REVIEW

File: app/components/dex-create.jsx [hunk 1]
```diff
@@ -72,6 +73,7 @@ export class DexCreate extends Component {
               <label htmlFor="generation">Generation</label>
               <select className="form-control" ref={(c) => this._generation = c} defaultValue="6">
```
Comment (ami) [inline app/components/dex-create.jsx]: should we default this to 7?

**Comment by ami** [comment]
@robinjoseph08 indicator added and region padding fixed!

File: app/components/dex-create.jsx (hunk 1 above)
Comment (robinjoseph08) [inline app/components/dex-create.jsx]: @ami should i also change the order from 6, 7 to 7, 6?

**Comment by robinjoseph08** [comment]
@ami i fixeded it!

**Comment by robinjoseph08** [comment]
@ami updated! wanna take one more look or is this good to merge?

**Comment by ami** [comment]
LGTM!

**Pull request merged**